plugin/completion/lib_complete.py

```diff
@@ -143,7 +147,8 @@ def parse_tu(self, view):
                     args=self.clang_flags,
                     unsaved_files=files,
                     options=TU.PARSE_PRECOMPILED_PREAMBLE |
-                    TU.PARSE_CACHE_COMPLETION_RESULTS)
+                    TU.PARSE_CACHE_COMPLETION_RESULTS |
+                    TU.PARSE_INCLUDE_BRIEF_COMMENTS_IN_CODE_COMPLETION)
```

Comment:
Libclang is quite picky about comments. This can be improved using raw comments and parsing them. For now I just added this flag as it is almost free functionality.
Try this snippet:

`/**
Comment about Foo
**/
extern void foo();


int main()
{
    foo();
}
`